src/provider/cloud-element-templates/ReplaceBehavior.js

```diff
@@ -33,7 +33,15 @@ export default class ReplaceBehavior extends CommandInterceptor {
         return;
       }
 
-      const { appliesTo } = elementTemplate;
+      const { appliesTo, elementType } = elementTemplate;
```

Comment:
If this closes https://github.com/bpmn-io/bpmn-js-properties-panel/issues/642, could you link the issue from the commit?